uffizzi/docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+x-uffizzi:
+  ingress:
+    service: umami
+    port: 3000
+
+services:
+  umami:
+    image: "${APP_IMAGE}"
+    ports:
+      - "3000:3000"
+    environment:
+      DATABASE_URL: postgresql://umami:umami@localhost:5432/umami
+      DATABASE_TYPE: postgresql
+      HASH_SALT: replace-me-with-a-random-string
```

Comment:
IF we want to show off, we can generate this and also the password within our compose file rendering step. But that's not required.

Reply:
I can do that, please let me know if I should do it.
My goal was to try to keep this `docker-compose` similar to original source.